src/tests/test_collection.c

```diff
@@ -165,29 +238,30 @@ MU_TEST(test_collection_load)
 {
   DLiteCollection *coll2;
   char *collpath = STRINGIFY(dlite_SOURCE_DIR) "/src/tests/coll.json";
-  DLiteStoragePathIter *iter;
-  const char *path;
+  //DLiteStoragePathIter *iter;
+  //const char *path;
 
   dlite_storage_paths_append(STRINGIFY(dlite_SOURCE_DIR) "/src/tests/*.json");
 
-  printf("\n\nStorage paths:\n");
-  iter = dlite_storage_paths_iter_start();
-  while ((path = dlite_storage_paths_iter_next(iter)))
-    printf("  - %s\n", path);
-  printf("\n");
-  dlite_storage_paths_iter_stop(iter);
+  //printf("\n\nStorage paths:\n");
+  //iter = dlite_storage_paths_iter_start();
+  //while ((path = dlite_storage_paths_iter_next(iter)))
+  //  printf("  - %s\n", path);
+  //printf("\n");
+  //dlite_storage_paths_iter_stop(iter);
 
   FILE *fp = fopen(collpath, "r");
   coll2 = (DLiteCollection *)
     dlite_json_fscan(fp, NULL, "http://onto-ns.com/meta/0.1/Collection");
   fclose(fp);
-  printf("\n\n--- coll2: %p ---\n", (void *)coll2);
-  dlite_json_print((DLiteInstance *)coll2);
-  printf("----------------------\n");
+  //printf("\n\n--- coll2: %p ---\n", (void *)coll2);
+  //dlite_json_print((DLiteInstance *)coll2);
+  //printf("----------------------\n");
+
   const DLiteInstance *inst = dlite_collection_get(coll2, "inst");
-  printf("\n--- inst: %p ---\n", (void *)inst);
-  dlite_json_print((DLiteInstance *)inst);
-  printf("----------------------\n");
+  //printf("\n--- inst: %p ---\n", (void *)inst);
+  //dlite_json_print((DLiteInstance *)inst);
+  //printf("----------------------\n");
 
```

Comment:
// can be removed altogether?

Reply:
Yes, they could. But loading is rather tricky involving loading a shared library via the plugin system and json-parsing, so it is not unlikely that we have to do some debugging in the future. Then these commented-out print statements comes in handy. I would therefore prefer to keep them.